aws/provider.go

```diff
@@ -233,6 +233,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_region":                           dataSourceAwsRegion(),
 			"aws_route":                            dataSourceAwsRoute(),
 			"aws_route_table":                      dataSourceAwsRouteTable(),
+			"aws_route_table_ids":                  dataSourceAwsRouteTableIDs(),
```

Comment:
Sorry this is going to seem real pedantic, but can you please:
* Rename the data source `aws_route_tables` along with the file names
* Rename the functions to match, e.g. `dataSourceAwsRouteTables()` and `dataSourceAwsRouteTablesRead()`

This is mainly for future maintainability as the data source may be adjusted to return information other than just IDs. We unfortunately have some other `_ids` data sources (e.g. `aws_subnet_ids`) that will likely need to be renamed at some point. 😅